### PR TITLE
docs: add faq non clustered operator

### PR DESCRIPTION
**Pull request by scrocquesel**

### CODE REVIEW

File: docs/documentation/faq.md [hunk 1]
```diff
@@ -43,4 +43,31 @@ without an update:
 ```
 
 Although you might consider using `EventSources`, to handle reconciliation triggering in a smarter
-way. 
\ No newline at end of file
+way. 
+
+### Q: How can I run an operator without cluster scope rights?
+
+By default, JOSDK require access to CR at cluster scope. You may not be granted such
```
Comment (metacosm) [inline docs/documentation/faq.md:50]:
```suggestion
By default, JOSDK requires access to CRDs at cluster scope. You may not be granted such
```

File: docs/documentation/faq.md [hunk 2]
```diff
@@ -43,4 +43,31 @@ without an update:
...
+at [Reconciler-level configuration](./configuration.md#reconciler-level-configuration):
+
+```java
+Operator operator;
+Reconciler reconciler;
+...
+operator.register(reconciler, configOverrider ->
+        configOverrider.settingNamespace("mynamespace"));
+```
+
```
Comment (metacosm) [inline docs/documentation/faq.md:67]:
```suggestion
Note that configuring the watched namespaces can also be done using the `@ControllerConfiguration` annotation.
```

**Review by metacosm — APPROVED** [review]

File: docs/documentation/faq.md [hunk 3]
```diff
@@ -43,4 +43,31 @@ without an update:
 ```
 
 Although you might consider using `EventSources`, to handle reconciliation triggering in a smarter
-way. 
\ No newline at end of file
+way. 
+
+### Q: How can I run an operator without cluster scope rights?
+
+By default, JOSDK requires access to CRDs at cluster scope. You may not be granted such
```
Comment (csviri) [inline docs/documentation/faq.md:50]: Is this true? By default we should not require access to CRDs just CRs.

**Review by csviri — CHANGES_REQUESTED** [review]

File: docs/documentation/faq.md (hunk 3 above)
Comment (metacosm) [reply docs/documentation/faq.md:50]: I guess, it's not true anymore but the javadoc hasn't been updated when the change was made, so it should be updated as well.

Comment (scrocquesel) [inline docs/documentation/faq.md:50]:
```suggestion
By default, JOSDK requires access to CRs at cluster scope. You may not be granted such
```

File: docs/documentation/faq.md [hunk 4]
```diff
@@ -43,4 +43,31 @@ without an update:
...
+
+```java
+Operator operator;
+Reconciler reconciler;
+...
+operator.register(reconciler, configOverrider ->
+        configOverrider.settingNamespace("mynamespace"));
+```
+Note that configuring the watched namespaces can also be done using the `@ControllerConfiguration` annotation.
+Furthermore, you may not be able to list CRDs at startup which is required when `checkingCRDAndValidateLocalModel`
```
Comment (scrocquesel) [inline docs/documentation/faq.md:68]:
```suggestion

Furthermore, you may not be able to list CRDs at startup which is required when `checkingCRDAndValidateLocalModel`
```

File: docs/documentation/faq.md [hunk 5]
```diff
@@ -43,4 +43,31 @@ without an update:
...
+is `true` (by default). To disable set it to `false` at [Operator-level configuration](./configuration.md#operator-level-configuration):
```
Comment (scrocquesel) [inline docs/documentation/faq.md:70]: @metacosm @csviri From the comment above, I understand the operator do not need "list CRD" right by default ? Does it mean `checkingCRDAndValidateLocalModel` is false by default now ?

Comment (csviri) [reply docs/documentation/faq.md:70]: yep the default should be false:
https://github.com/java-operator-sdk/java-operator-sdk/blob/a6299203025f7e4237cd32f43a4308dd66f7e2ac/operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/ConfigurationService.java#L68-L68

**Review by csviri — APPROVED** [review]

**Review by metacosm — APPROVED** [review]

**Pull request merged**